src/components/Journal.js

```diff
@@ -1,10 +1,14 @@
 import React, { Component } from 'react'
+import Button from './Button'
 
 class Journal extends Component {
   render() {
     return (
       <div>
-        <h3>This is your journal!</h3>
```

Comment:
I moved this to Containers, since it will have a decent amount of logic in it, plus many Read Components